tests/testthat/test-api-workflows.R

```diff
@@ -30,41 +30,66 @@ test_that("can get workflow summary", {
     ref = scalar(ref)
   )), t)
 
-  output <- list(
+  workflow <- list(
     reports = list(
       list(
-        name = scalar("preprocess"),
-        instance = scalar("production"),
+        name = "preprocess",
+        instance = "production",
         params = list(
-          nmin = scalar(0.5),
-          nmax = scalar(2)
+          nmin = 0.5,
+          nmax = 2
         ),
-        depends_on = list()
+        depends_on = c()
       ),
       list(
-        name = scalar("process"),
-        instance = scalar("production"),
-        depends_on = list(scalar("preprocess"))
+        name = "process",
+        instance = "production",
+        depends_on = c("preprocess")
       ),
       list(
-        name = scalar("postprocess"),
-        instance = scalar("production"),
-        depends_on = list(scalar("preprocess"), scalar("process"))
+        name = "postprocess",
+        instance = "production",
+        depends_on = c("preprocess", "process")
       )
     ),
-    ref = scalar("123"),
+    ref = "123",
     missing_dependencies = list(
-      preprocess = "",
-      process = "",
-      postprocess = ""
+      preprocess = list(),
```

Comment:
I think good we did this slight testing change as it has shown areas where the data also did not match what the function would actually return